tests/parametric/test_tracer_flare.py

```diff
@@ -0,0 +1,147 @@
+"""
+Test the tracer flare feature of the APM libraries.
+"""
+import json
+import zipfile
+from base64 import b64decode
+from io import BytesIO
+from typing import Any
+from typing import Dict
+from uuid import uuid4
+
+import pytest
+
+from utils import context, missing_feature, rfc, scenarios
+
+parametrize = pytest.mark.parametrize
+
+TEST_SERVICE = "test_service"
+TEST_ENV = "test_env"
+DEFAULT_ENVVARS = {
+    "DD_SERVICE": TEST_SERVICE,
+    "DD_ENV": TEST_ENV,
+    # Needed for .NET until Telemetry V2 is released
+    "DD_INTERNAL_TELEMETRY_V2_ENABLED": "1",
+    # Decrease the heartbeat/poll intervals to speed up the tests
+    "DD_TELEMETRY_HEARTBEAT_INTERVAL": "0.2",
+    "DD_REMOTE_CONFIG_POLL_INTERVAL_SECONDS": "0.2",
+}
+
+
+def _tracer_flare_task_config() -> Dict[str, Any]:
+    return {
+        "args": {"case_id": "12345", "hostname": "my.hostname", "user_handle": "its.me@datadoghq.com"},
+        "task_type": "tracer_flare",
+    }
+
+
+def _flare_log_level_order() -> Dict[str, Any]:
+    return {
+        "order": [],
+        "internal_order": [
+            "flare-log-level.trace",
+            "flare-log-level.debug",
+            "flare-log-level.info",
+            "flare-log-level.warn",
+            "flare-log-level.error",
+            "flare-log-level.critical",
+            "flare-log-level.off",
+        ],
+    }
+
+
+def _set_log_level(test_agent, log_level: str) -> None:
+    """Helper to create the appropriate "flare-log-level" config in RC for a given log-level.
+    """
+    cfg_id = "flare-log-level.%s" % log_level
+    test_agent.set_remote_config(
+        path="datadog/2/AGENT_CONFIG/%s/config" % cfg_id, payload={"name": cfg_id, "config": {"log_level": log_level}}
```

Comment:
We commonly use f-strings on system tests : 

```suggestion
        path=f"datadog/2/AGENT_CONFIG/{cfg_id}/config", payload={"name": cfg_id, "config": {"log_level": log_level}}
```

There are others, I don't add a comment for each ofthem 😅

Reply:
thanks, I'll change them all to use f-strings

Reply:
63b238adde4a1473689e5598fb7043d3592505e0